mappings/net/minecraft/entity/ai/brain/EntityMemoryModule.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS anw net/minecraft/entity/ai/brain/EntityMemoryModule
```

Comment:
The vanilla name is MemoryModule**Type** (based on name in `Registry`). It doesn't actually store any data, it's used as a key to get the memory module.